app/src/androidTest/java/org/simple/clinic/storage/DeleteSyncGroupDatabaseAndroidTest.kt

```diff
@@ -27,6 +27,7 @@ class DeleteSyncGroupDatabaseAndroidTest {
   private val bloodPressureMeasurementDao by lazy { database.bloodPressureDao() }
```

Comment:
There is a typo in the commit message, "madical -> medical"

Reply:
Good catch! Will fix!